### ISSUE TITLE
support for secondary_y in df.iplot()

### ISSUE THREAD

**Issue by jackparmer** [santosjorge/cufflinks#12]
This would be awesome too. Setting asFigure=True and adding the axis in Plotly's native syntax is a little clunky.
http://pandas.pydata.org/pandas-docs/version/0.15.0/visualization.html#plotting-on-a-secondary-y-axis

**Comment by santosjorge** [comment]
You can do that now with 3 steps:
```python
fig=cf.datagen.lines(3,columns=['a','b','c']).figure()
fig=fig.set_axis('b',side='right')
cf.iplot(fig)
```

However I agree it can be done even easier. I will add `secondary_y` support as a parameter to iplot.

**Comment by santosjorge** [comment]
Fixed in v0.6
